modules/apps/transfer/types/trace.go

```diff
@@ -156,7 +178,13 @@ func ValidatePrefixedDenom(denom string) error {
 		return sdkerrors.Wrap(ErrInvalidDenomForTransfer, "base denomination cannot be blank")
 	}
 
-	identifiers := denomSplit[:len(denomSplit)-1]
+	path, _ := extractPathAndBaseDenomFromDenomTrace(denomSplit)
```

Comment:
How come the base denom arg is ignored? Should we add a comment? Or is the second return value needed?

Reply:
In this case the base denom is not needed at all because the only validation that happens here is for the trace path information.